packages/client/README.md

```diff
@@ -10,97 +10,188 @@
 
 [![Build status](https://github.com/streamr-dev/monorepo/actions/workflows/client-build.yml/badge.svg)](https://github.com/streamr-dev/monorepo/actions/workflows/client-build.yml)
 ![latest npm package version](https://img.shields.io/npm/v/streamr-client?label=latest)
-[![GitHub stars](https://img.shields.io/github/stars/streamr-dev/network-monorepo?style=social)
+![GitHub stars](https://img.shields.io/github/stars/streamr-dev/network-monorepo?style=social)
 [![Discord Chat](https://img.shields.io/discord/801574432350928907.svg?label=Discord&logo=Discord&colorB=7289da)](https://discord.gg/FVtAph9cvz)
 
 This library allows you to easily interact with the [Streamr Network](https://streamr.network) from JavaScript-based environments, such as browsers and [node.js](https://nodejs.org). The library wraps a Streamr light node for publishing and subscribing to data, as well as contains convenience functions for creating and managing streams.
```

Comment:
We should consider if we refer to message (preferably) or data, and not use both words in this document?

Reply:
What does this make reference to?

Reply:
I mean that in this document we should use only one word to reference to the input type of a subscription (and also output type of a publish, etc). Usually we use word "message", but sometimes e.g. word "data", like here. 

I suggest that we'd update this sentence to be:
"...publishing and subscribing messages..."

And also all other occurrences in the document, e.g.
" ..."By default `subscribe` will not resend historical messages. It only subscribes to real time messages..." (line 222)

(If some sentence are bit boring after that change it could make sense to shorten those. Not a high priority thing.)